macros/StatusVmon904HV.py

```diff
@@ -0,0 +1,1339 @@
+#!/usr/bin/env python
```

Comment:
So in this file and also the one below it I would suggest to make a few changes that we discussed on mattermost [here](https://mattermost.web.cern.ch/cms-gem-ops/pl/1pu6j13jtfy9up7kjst9etczbo) still.

Instead of using `raw_input` you can build a help menu and also have command line options or required arguments using `argparse.ArgumentParser`.  And we should encapsulate the below in a `main`.